docs/getting_started_guide/register.rst

```diff
@@ -36,9 +36,18 @@ to each provider supported by ``eodag``:
   Then go to your profile and use from the section named "API key" the *UID* as *username* and *API Key* as *password* in eodag credentials.
   EODAG can be used to request for public datasets, you can browse them `here <https://ads.atmosphere.copernicus.eu/cdsapp#!/search?type=dataset>`__.
 
-* ``cop_cds``: create an account `here <https://cds.climate.copernicus.eu/user/register>`__.
-  Then go to your profile and use from the section named "API key" use *UID* as *username* and *API Key* as *password* in eodag credentials.
-  EODAG can be used to request for public datasets, you can browse them `here <https://cds.climate.copernicus.eu/cdsapp#!/search?type=dataset>`__.
+* ``cop_cds``: create an ecmwf account `here <https://accounts.ecmwf.int/auth/realms/ecmwf/login-actions/registration?client_id=cms-www&tab_id=1TPxB0N2QVA>`__.
```

Comment:
The page returns the error `Cookie not found. Please make sure cookies are enabled in your browser.`

I suggest using the link `https://cds-beta.climate.copernicus.eu/` or `https://www.ecmwf.int/` and then adding the instructions about where to click to register.

Reply:
changed